lib/drivex.js

```diff
@@ -75,6 +75,26 @@ DriveX.prototype = {
       }
     });
   },
+    /**
+   * wraps Selenium !WebElement.isDisplayed()
+   * @param locator {LocatorJSON}
+   * @param el {WebElement}
+   * @returns {Promise} resolves to true or rejected
+   */
+  notVisible: function (locator, el) {
+    var self = this;
+    return (el ? el : this.driver).isElementPresent(this.by(locator)).then(function(isPresent) {
+      if (isPresent) {
+        return self.find(locator, el).then(function(elt) {
+          return elt.isDisplayed().then(function(v) {
+            return !v;
+          });
+        })
+      } else {
+        return false;
+      }
+    });
+  },
```

Comment:
Didn't realize earlier that there was `visible` method. You can just use it instead of duplicating the code here, like
```
return self.visible(locator).then(function(isVisible) {
  return !isVisible
});
```